spec/rails_helper.rb

```diff
@@ -92,7 +92,8 @@
   config.cassette_library_dir = "spec/fixtures/vcr"
   config.hook_into :webmock
   config.configure_rspec_metadata!
-  config.ignore_hosts "localhost", "127.0.0.1"
+  config.ignore_localhost = true
```

Comment:
`127.0.0.1` was part of our webmock config and I did have some failing tests without it - maybe the move to request specs has ironed that out!

Reply:
It hasn't, as we still use webmock in system specs.